super_diff.gemspec

```diff
@@ -12,7 +12,7 @@ Gem::Specification.new do |s|
     SuperDiff is a gem that hooks into RSpec to intelligently display the
     differences between two data structures of any type.
   DESC
-  s.required_ruby_version = "~> 2.4"
+  s.required_ruby_version = ">= 2.4"
```

Comment:
Even if SuperDiff hasn't hit 1.0 yet, I think we ought to be conservative here and say `[">= 2.4", "< 4.0"]`. What do you think?